spec/models/lettings_log_spec.rb

```diff
@@ -2493,40 +2493,69 @@ def previous_postcode_fields
     let(:user) { FactoryBot.create(:user, organisation: location.scheme.owning_organisation) }
     let(:expected_content) { csv_export_file.read }
 
-    before do
-      Timecop.freeze(Time.utc(2022, 6, 5))
-      lettings_log = FactoryBot.create(:lettings_log, needstype: 2, scheme:, location:, owning_organisation: scheme.owning_organisation, created_by: user, rent_type: 2, startdate: Time.zone.local(2021, 10, 2))
-      expected_content.sub!(/\{id\}/, lettings_log["id"].to_s)
-      expected_content.sub!(/\{scheme_code\}/, "S#{scheme['id']}")
-      expected_content.sub!(/\{scheme_service_name\}/, scheme["service_name"].to_s)
-      expected_content.sub!(/\{scheme_sensitive\}/, scheme["sensitive"].to_s)
-      expected_content.sub!(/\{scheme_primary_client_group\}/, scheme["primary_client_group"].to_s)
-      expected_content.sub!(/\{scheme_secondary_client_group\}/, scheme["secondary_client_group"].to_s)
-      expected_content.sub!(/\{scheme_support_type\}/, scheme["support_type"].to_s)
-      expected_content.sub!(/\{scheme_intended_stay\}/, scheme["intended_stay"].to_s)
-      expected_content.sub!(/\{location_code\}/, location["id"].to_s)
-      expected_content.sub!(/\{location_startdate\}/, location["startdate"].to_s)
-      expected_content.sub!(/\{scheme_id\}/, scheme["service_name"].to_s)
-      expected_content.sub!(/\{location_id\}/, location["id"].to_s)
-    end
-
     after do
       Timecop.unfreeze
     end
 
-    context "with a support user" do
-      let(:csv_export_file) { File.open("spec/fixtures/files/lettings_logs_download.csv", "r:UTF-8") }
+    context "with values represented as human readable labels" do
+      before do
+        Timecop.freeze(Time.utc(2022, 6, 5))
+        lettings_log = FactoryBot.create(:lettings_log, needstype: 2, scheme:, location:, owning_organisation: scheme.owning_organisation, created_by: user, rent_type: 2, startdate: Time.zone.local(2021, 10, 2))
+        expected_content.sub!(/\{id\}/, lettings_log["id"].to_s)
+        expected_content.sub!(/\{scheme_code\}/, "S#{scheme['id']}")
+        expected_content.sub!(/\{scheme_service_name\}/, scheme["service_name"].to_s)
+        expected_content.sub!(/\{scheme_sensitive\}/, scheme["sensitive"].to_s)
+        expected_content.sub!(/\{scheme_primary_client_group\}/, scheme["primary_client_group"].to_s)
+        expected_content.sub!(/\{scheme_secondary_client_group\}/, scheme["secondary_client_group"].to_s)
+        expected_content.sub!(/\{scheme_support_type\}/, scheme["support_type"].to_s)
+        expected_content.sub!(/\{scheme_intended_stay\}/, scheme["intended_stay"].to_s)
+        expected_content.sub!(/\{location_code\}/, location["id"].to_s)
+        expected_content.sub!(/\{location_startdate\}/, location["startdate"].to_s)
+        expected_content.sub!(/\{scheme_id\}/, scheme["service_name"].to_s)
+        expected_content.sub!(/\{location_id\}/, location["id"].to_s)
+        expected_content.gsub!("\r\n", "\n")
```

Comment:
- [ ] I should discuss this and the fact that I am calling strip on everything below as I shouldn't need to have a fight with line endings. I found myself unable to force the pooter to save these csv files with LF line endings

Reply:
this has resolved itself